Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:jammy
+
+LABEL py_version='3.x'
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+VOLUME /dist
+
+RUN apt-get update && apt-get install -y \
+    curl \
+    libgdal-dev \
+    libhdf5-dev \
+    libnetcdf-dev \
+    libxml2-dev \
+    python3-dev \
+    python3-venv \
+    g++ \
+    cmake \
+    git \
+    unzip \
+    zip \
+&& rm -rf /var/lib/apt/lists/*
+
+# First fetch the code for MDAL and mdal-python
+
+WORKDIR /
+
+ARG MDALVERSION=release-1.0.3
```

Comment:
I'd prefer leaving the `release-` prefix out; so that you can just provide a version

Reply:
I had that at first, but looking at https://github.com/lutraconsulting/MDAL/tags, some of the tags contain the word "release" and some don't. It's slightly annoying to have to type "release" in the MDALVERSION arg, but it would be more annoying not to be able to specify a certain MDAL version.